File: src/main/java/com/tinny/commons/helper/AppLogger.java
package com.tinny.commons.helper;


import android.util.Log;


/**
 * This class use to create the application logs
 */


public class AppLogger {
    public static void debugLogs(String TAG, String message) {
        Log.d(TAG, message);
    }

    public static void errorLogs(String TAG, String message) {
        Log.e(TAG, message);

    }


}

File: src/main/java/com/tinny/commons/helper/AppLogger.kt
package com.tinny.commons.helper

import android.util.Log

/**
 * This class use to create the application logs
 */
object AppLogger {
    fun debugLogs(TAG: String = "Test===", message: String) {
        Log.d(TAG, message)
    }

    fun errorLogs(TAG: String?, message: String?) {
        Log.e(TAG, message!!)
    }

}